process_jartic_simple_v2.py

```
@@ -0,0 +1,205 @@
+import asyncio
+from datetime import datetime, timezone
+from pathlib import Path
+import zipfile
+import csv
+from tqdm import tqdm
+import io
+
+from src.core.data_storage import DataStorage
+
+
+def count_total_measurements(archives: list, start_date: datetime, end_date: datetime) -> int:
```

Ruff (0.12.2)

Unused function argument: `start_date`

(ARG001)